recipes/sk-dist/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sk-dist" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd956610ef88343686c6dcc1e01df617829cdf43444a4d3a41437b214df8a89d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # - {{ compiler('c') }}
+
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - joblib
+    - numpy
+    - pandas >=0.17.0
+    - python >=3.5
+    - scikit-learn >=0.20.0
+    - scipy
+
+test:
+  imports:
+    - skdist
+    - skdist.distribute
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/sk-dist/
+  summary: "Distributed scikit-learn meta-estimators with PySpark"
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
It looks like the license file is not included in the source distribution. I've submitted a PR to fix this (https://github.com/Ibotta/sk-dist/pull/47). Until that gets merged or another solution is implemented upstream, you will need to manually package the license file. See:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Also please add the following comment as a visual reminder to check the status of the license file in the source distribution in future updates to the feedstock:

```yaml
# License file manually packaged. See https://github.com/Ibotta/sk-dist/pull/47
license_file: LICENSE
...
```

When the license file is included in the source distribution, you can delete this comment and remove the manually packaged copy.

Reply:
PR you submitted was already merged. No changes should be needed then.

Reply:
The license file won't be in the source distribution until a new release is pushed to pypi. Until then, leave the comment in as a reminder and manually package the license file.